### DISCUSSION TITLE
Bug Report: Selected Vector Transformation Undo Resizing Bug

### DISCUSSION THREAD

**Discussion by miya-sans** [keiwando/toonsquid#329] [Q&A] [answered]
Device: iPad Pro (12.9 inch) (6th generation)
iOS Version: 16.4.1
Version: 1.9

https://github.com/keiwando/toonsquid/assets/42623210/411cf320-e1a4-466f-8a0d-79e74045426c

Steps to recreate:
1. Draw a vector line
2. Select part of the line
3. Transform Resize twice
4. Undo twice
7. Transform move
8. Line will resize back to before the undo

Possibly related to the fix for https://github.com/keiwando/toonsquid/discussions/308

Does not affect raster layers, unselected transformations for vector and raster layers.

**Comment by keiwando** [comment, marked as answer]
This is now fixed with the 1.9.1 update. Thanks for the report!
